lms/templates/instructor/instructor_dashboard_2/extensions.html

```diff
@@ -1,8 +1,11 @@
-<%! from django.utils.translation import ugettext as _ %>
-<%page args="section_data"/>
+<%page args="section_data" expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
```

Comment:
Don't import things you aren't using. I don't see any references to HTML or Text.